docs/content/latest/contribute/_index.md

```diff
@@ -10,7 +10,7 @@ section: CONTRIBUTOR GUIDES
 menu:
   latest:
     identifier: contribute
-    weight: 2900
+    weight: 2800
```

Comment:
the CLA section should point to https://cla-assistant.io/yugabyte/yugabyte-db instead of the google doc.

Reply:
Fixed